openupgrade_scripts/scripts/analytic/16.0.1.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,94 @@
+---Models in module 'analytic'---
+obsolete model account.analytic.distribution
+obsolete model account.analytic.tag
+new model account.analytic.applicability
+new model account.analytic.distribution.model
+new model analytic.mixin [abstract]
+# NOTHING TO DO
+
+obsolete model account.analytic.group (renamed to account.analytic.plan)
+new model account.analytic.plan (renamed from account.analytic.group)
+# DONE pre-migration: rename model
+
+---Fields in module 'analytic'---
+analytic     / account.analytic.account / _order                        : _order is now 'plan_id, name asc' ('code, name asc')
+# NOTHING TO DO
+
+analytic     / account.analytic.account / group_id (many2one)           : DEL relation: account.analytic.group
+analytic     / account.analytic.account / plan_id (many2one)            : NEW relation: account.analytic.plan, required
+# DONE pre-migration: rename field and fill value for null row
+
+analytic     / account.analytic.account / root_plan_id (many2one)       : NEW relation: account.analytic.plan, isfunction: function, stored
+# DONE pre-migration: create column and pre-fill value to avoid computed by ORM
+
+analytic     / account.analytic.applicability / analytic_plan_id (many2one)   : NEW relation: account.analytic.plan
+analytic     / account.analytic.applicability / applicability (selection)     : NEW required, selection_keys: ['mandatory', 'optional', 'unavailable']
+analytic     / account.analytic.applicability / business_domain (selection)   : NEW required, selection_keys: ['general']
```

Comment:
I think this one and the one i comment above should be handled in `account` module